src/main/java/com/julienvey/trello/impl/TrelloImpl.java

```diff
@@ -103,19 +110,14 @@ public TrelloImpl(String applicationKey, String accessToken, TrelloHttpClient ht
     public Board getBoard(String boardId, Argument... args) {
         Board board = get(createUrl(GET_BOARD).params(args).asString(), Board.class, boardId);
         board.setInternalTrello(this);
-        for (TList list : board.getLists()) {
-            list.setInternalTrello(this);
-        }
+        setInternalTrello(board.getLists());
```

Comment:
let's just do something like 

    members.forEach(m -> m.setInternalTrello(this));

using Java 8 syntax

Reply:
I saw lines in the `build.gradle`
```
sourceCompatibility = 1.7
targetCompatibility = 1.7
```

Reply:
I saw that you changed `1.7` to `1.8` so I would provide more Stream oriented solution